src/org/omegat/core/search/Searcher.java

```diff
@@ -174,9 +174,15 @@ public void search() throws TranslationException, PatternSyntaxException, IOExce
         switch (expression.searchExpressionType) {
         case EXACT:
```

Comment:
> A user who use REGEXP may want to control search behavior by itself. 
You can make the same argument about the case-sensitivity option, but that works for all search modes. And the user can have full control by simply turning the sensitivity-related options off.

> I'd like to make option disable when selecting REGEXP.
I consider having this work in all modes to be a requirement for accepting this pull request.